lib/much-rails/models/not_destroyable.rb

```diff
@@ -0,0 +1,27 @@
+require "much-plugin"
+require "much-rails/models/validate_destroy"
+
+# MuchRails::Models::NotDestroyable is a mix-in to disable destroying a
+# domain model. It mixes-in MuchRails::Models::ValidateDestroy and hard-codes
+# a permanent destruction error message.
+module MuchRails; end
+module MuchRails::Models; end
+module MuchRails::Models::NotDestroyable
+  include MuchPlugin
+
+  plugin_included do
+    include MuchRails::Models::ValidateDestroy
+  end
+
+  plugin_instance_methods do
+    def destruction_error_messages
+      ["#{self.class.humanized_name.pluralize} can't be deleted."]
```

Comment:
```suggestion
      ["#{self.class_name} records can't be deleted."]
```

This should work, can you verify? We don't have the dumb humanized name stuff here and if we do b/c Rails, I don't want to couple to it.

Reply:
we can do `self.class.name`, but not `self.class_name`...cool?

Reply:
Cool!